benchmark/results.txt

```diff
@@ -0,0 +1,8 @@
+\begin{tabular}{lrr}
+\hline
+ Model     &   MNIST &   Iris \\
+\hline
+ SparseMax &   0.050 &  0.040 \\
```

Comment:
Can you transpose this, so it is like `timings.txt` I also think the confidence intervals are missing.